app-server/app/src/test/scala/footprint/FootprintSpec.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.footprint
+
+import org.scalatest.{Matchers, WordSpec}
+import akka.http.scaladsl.testkit.{ScalatestRouteTest, RouteTestTimeout}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.model.{HttpEntity, ContentTypes}
+import akka.actor.ActorSystem
+import concurrent.duration._
+import spray.json._
+
+import com.azavea.rf.utils.Config
+import com.azavea.rf.{DBSpec, Router}
+import com.azavea.rf.datamodel.latest.schema.tables._
+import com.azavea.rf.utils.PaginatedResponse
+import com.azavea.rf.AuthUtils
+
+class FootprintSpec extends WordSpec
+    with Matchers
+    with ScalatestRouteTest
+    with Config
+    with Router
+    with DBSpec {
+  implicit val ec = system.dispatcher
+  implicit def database = db
+  implicit def default(implicit system: ActorSystem) = RouteTestTimeout(DurationInt(20).second)
+
+  val authorization = AuthUtils.generateAuthHeader("Default")
+  "/api/footprints" should {
+    "return a paginated list of footprints" in {
+      Get("/api/footprints") ~> footprintRoutes ~> check {
+        responseAs[PaginatedResponse[FootprintsRow]]
+      }
+    }
+    "allow the creation of new footprints" in {
+    }
```

Comment:
Looks like this is missing some tests -- would be good to see the bounding box query parameter tested as well